<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>SelectDialog</class>
    <widget class="QDialog" name="SelectDialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>250</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Select Players</string>
        </property>

        <!-- Layout contenant les éléments -->
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <widget class="QCheckBox" name="player1CheckBox">
                    <property name="text">
                        <string>Player 1 Ready</string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QCheckBox" name="player2CheckBox">
                    <property name="text">
                        <string>Player 2 Ready</string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QLabel" name="statusLabel">
                    <property name="text">
                        <string>Waiting for both players to be ready...</string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QPushButton" name="startButton">
                    <property name="text">
                        <string>Start Game</string>
                    </property>
                </widget>
            </item>
        </layout>

    </widget>
</ui>
